app/assets/stylesheets/govuk_publishing_components/components/_table.scss

```diff
@@ -125,3 +125,39 @@ $table-row-even-background-colour: govuk-colour("light-grey");
     border: 0;
   }
 }
+
+// stylelint-disable declaration-no-important
+@include govuk-media-query($media-type: print) {
+  .govuk-table--sortable {
+    outline: none;
+
+    * {
+      color: $govuk-print-text-colour !important;
+      background-color: initial !important;
```

Comment:
Is the `important` needed here? Just tested it without and it seemed okay.

Reply:
I'm not seeing the same thing: if I remove `!important` then the old blue/inverted headings reappear and the zebra striping on the rows re-appears. (This is when the browser is set to print background graphics, did you test it that way?)

Reply:
Ah yes, I did not have the option to print background graphics enabled, fair enough.